python/cuml/__init__.py

```diff
@@ -1,5 +1,5 @@
 #
-# Copyright (c) 2019, NVIDIA CORPORATION.
+# Copyright (c) 2020, NVIDIA CORPORATION.
```

Comment:
```suggestion
# Copyright (c) 2019-2020, NVIDIA CORPORATION.
```

Reply:
Should this not be from either 2017 or 2018 to 2020?

Reply:
no